### DISCUSSION TITLE
Network devices ConfGen/ConfTemplates Netbox plugin

### DISCUSSION THREAD

**Discussion by dmulyalin** [netbox-community/netbox#12060] [Ideas]
Hi, are there any interest in having plugin to generate devices configurations using templates as part of Netbox? As well as to store (Jinja2) templates inside of the Netbox. With capability to attach templates to entities inside of the Netbox e.g. attach a list of templates to services or to devices or to interfaces etc.

**Comment by kkthxbye-code** [comment]
Fyi, this is being added natively in netbox 3.5:

https://github.com/netbox-community/netbox/blob/feature/docs/release-notes/version-3.5.md#configuration-template-rendering-11559

**Comment by dmulyalin** [comment]
Thank you for the pointer :)
